### ISSUE TITLE
remove docs directory

### ISSUE THREAD

**Issue by xmonader** [threefoldtech/info_grid#173]
remove all generated files from the repo, the workflow on the remote machines needs to trigger the build

**Comment by xmonader** [comment]
@Mik-TF @Mahmoud-Emad please see the best path forward

**Comment by Mik-TF** [comment]
@xmonader 
I removed the docs folder and did a commit.

The manual (www2.manual.grid.tf) is OK it seems, but we get "pages build and deployment" error.

Maybe we'd need to change the jobs process [here](https://github.com/threefoldtech/info_grid/actions/runs/5475631870/jobs/9972002554).

It might just be the workflow that isn't yet updated correctly, since the deployment seems to work fine, see [here](https://github.com/threefoldtech/info_grid/actions/runs/5475631932/jobs/9972002035#step:3:12).

Please let me know what you think.

**Comment by Mik-TF** [comment]
Maybe the easiest solution is just to have the docs still there, but make suer the .gitignore docs/ and book/ are working correctly.

**Comment by Mik-TF** [comment]
After some testings, we really do need to remove the docs folder and update the workflow in other for the mdbook to be built entirely on the web-server.

In short, I deleted docs in dev branch, but then it didn't complete the build process, see [here](https://github.com/threefoldtech/info_grid/commits/development).

- 77b379054609b94de906a91fca5d6e137eacd94c
- 3513fe843299ba065d56928724779c8f3f332711

***

When cloning the current dev branch locally, and running mdbook serve..., everything works 100%. So we really just need to remove the docs files and folder, and update the workflow.

***

As discussed with @Mahmoud-Emad, he will be working on this issue.
Thanks Emad!

**Comment by Mik-TF** [comment]
I set this as critical, since the new changes in the manual aren't shown on the website www2.manual.grid.tf until we fix this.

**Comment by Mahmoud-Emad** [comment]
After thorough investigation, I discovered that the issue with the Github pages, as you mentioned earlier, occurred due to the pages workflow searching for the docs directory. Since we have removed the docs directory from the development branch, the pages are no longer necessary.

As a next step, I will disable the pages workflow and retest the changes I made. This will allow me to verify if any of the modifications have caused any unforeseen issues.
